domains/transfer/Transfer.API/API.IntegrationTests/Cvr/Api/v2024_01_03/Controllers/CvrMultipleResponseTests.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http.Json;
+using System.Threading.Tasks;
+using API.Cvr.Api.v2024_01_03.Dto.Responses;
+using API.IntegrationTests.Factories;
+using FluentAssertions;
+using VerifyTests;
+using VerifyXunit;
+using WireMock.RequestBuilders;
+using WireMock.ResponseBuilders;
+using WireMock.Server;
+using Xunit;
+
+namespace API.IntegrationTests.Cvr.Api.v2024_01_03.Controllers;
+//This should be in the same class as CvrControllerTests, but WireMock does not Co-operate with that.
```

Comment:
Why is that?

Reply:
Tests, using wiremock, share resources, so the test always fail when running in parallel with the others. this is the fix suggested by @tnickelsen

Reply:
Please put this explanation as a comment

Reply:
I've put a comment in the top of the PR

Reply:
Sorry for not being clear. I ment to be more specific in the existing comment in the code on why it is not possible

Reply:
Done:

https://github.com/Energinet-DataHub/energy-origin/pull/558/commits/e9a7d13a9eb6f54fb7dc358ac40b449cba881328